pkg/scheduler/framework/plugins/nodevolumelimits/non_csi_test.go

```diff
@@ -74,6 +74,12 @@ var (
 	anotherUnboundPVCPod = st.MakePod().PVC("anotherUnboundPVC").Obj()
 )
 
+var cmpOpts = []cmp.Option{
+	cmp.Comparer(func(a, b *framework.Status) bool {
+		return reflect.DeepEqual(a, b)
```

Comment:
this defies the purpose of migrating to `cmp.Diff`

We already have a comparer for Status, but it requires that errors satisfy `errors.Is`. To accomplish this, errors should either implement `Is` or you have to use the same error pointer in the test data.

Reply:
Thanks, I will fix it.